modules/installation-configure-proxy.adoc

```diff
@@ -62,6 +65,17 @@ range that is specified in the `networking.machineNetwork[].cidr` field in the
 ====
 endif::bare-metal[]
 
+ifdef::kuryr[]
+[NOTE]
+====
+For Kuryr installations on restricted networks that use the `Proxy` object, the proxy must be able to reply to router of the {product-title} cluster. To add a static route for the proxy configuration:
```

Comment:
```suggestion
For Kuryr installations on restricted networks that use the `Proxy` object, the proxy must be able to reply to the router of the {product-title} cluster. To add a static route for the proxy configuration:
```

Could "the proxy must be able to reply to the router of the {product-title} cluster" become "the proxy must be able to reply to the router that the cluster uses"? @MaysaMacedo

Reply:
yes.